engine/src/test/java/edu/kit/algorithms/or1/NorthWestTest.java

```diff
@@ -0,0 +1,47 @@
+package edu.kit.algorithms.or1;
+
+import edu.kit.algorithms.or1.model.TransportProblem;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class NorthWestTest {
+    private TransportProblem transportProblem;
+
+    @Test
+    void solve1() {
+        int[][] costs = {{7, 2, 4, 7}, {9, 5, 3, 3}, {7, 7, 6, 4}};
```

Comment:
Maybe in ne Methode auslagern, die das TransportProblem erstellt. Dann kann man sich Code sparen für Vogel